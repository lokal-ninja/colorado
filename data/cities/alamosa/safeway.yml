id: 401538196
type: way
latitude: 37.47004
longitude: -105.87702
postcode: "81101"
city: Alamosa
street: "Main Street"
housenumber: 1301
phone: "+1 719-587-3075"
opening_hours: ""
website: "https://local.safeway.com/safeway/co/alamosa/1301-main-st.html"