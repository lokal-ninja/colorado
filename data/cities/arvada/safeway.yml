id: 210073964
type: node
latitude: 39.8438
longitude: -105.08134
postcode: "80003"
city: Arvada
street: West 80th Avenue
housenumber: "7561"
phone: +1 303-420-0318
opening_hours: Mo-Su 06:00-23:00
website: https://local.safeway.com/safeway/co/arvada/7561-w-80th-ave.html
