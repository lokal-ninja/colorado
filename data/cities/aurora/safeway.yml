id: 1271583342
type: node
latitude: 39.65128
longitude: -104.76961
postcode: "80013"
city: Aurora
street: "East Hampden Avenue"
housenumber: 18730
phone: "+1-720-876-2244"
opening_hours: "Su-Sa 05:00-00:00"
website: "https://local.safeway.com/safeway/co/aurora/18730-e-hampden-ave.html"