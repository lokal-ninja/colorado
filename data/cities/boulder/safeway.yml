id: 106152735
type: way
latitude: 40.01349
longitude: -105.26015
postcode: "80302"
city: Boulder
street: "Arapahoe Avenue"
housenumber: 2798
phone: ""
opening_hours: "Mo-Su 06:00-23:00"
website: "https://local.safeway.com/safeway/co/boulder/2798-arapahoe-ave.html"
