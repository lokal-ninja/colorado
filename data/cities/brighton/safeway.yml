id: 512454968
type: way
latitude: 39.98809
longitude: -104.80205
postcode: "80601"
city: Brighton
street: "East Bridge Street"
housenumber: 1605
phone: ""
opening_hours: ""
website: ""