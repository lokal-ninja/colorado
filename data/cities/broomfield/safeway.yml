id: 233062500
type: way
latitude: 39.9568
longitude: -105.03601
postcode: "80023"
city: Broomfield
street: "West 144th Avenue"
housenumber: 3600
phone: "+1 303-209-2405"
opening_hours: "Mo-Su 05:00-12:00"
website: "https://local.safeway.com/safeway/co/broomfield/3602-w-144th-ave.html"