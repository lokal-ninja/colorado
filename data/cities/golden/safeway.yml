id: 92727524
type: way
latitude: 39.75248
longitude: -105.21615
postcode: "80401"
city: Golden
street: "Jackson Street"
housenumber: 1701
phone: ""
opening_hours: ""
website: ""